### PR TITLE
fix: export a "isResponseWithoutBody" utility

### PR DESCRIPTION
**Pull request by kettanaito**
- Related to https://github.com/mswjs/msw/pull/1894

### CODE REVIEW

File: src/utils/responseUtils.ts [hunk 1]
```diff
@@ -2,4 +2,12 @@
  * Response status codes for responses that cannot have body.
  * @see https://fetch.spec.whatwg.org/#statuses
  */
-export const responseStatusCodesWithoutBody = [204, 205, 304]
+export const RESPONSE_STATUS_CODES_WITHOUT_BODY = [101, 103, 204, 205, 304]
```
Comment (mattcosta7) [inline src/utils/responseUtils.ts:5]: super duper nitty:

Should we make this a set instead of an array? super minimal but a bit more semantic

**Review by mattcosta7 — APPROVED** [review]

File: src/utils/responseUtils.ts (hunk 1 above)
Comment (mattcosta7) [reply src/utils/responseUtils.ts:5]: CI seems to suggest we throw on 101/103, which I think makes sense.

101 is switching protocols (changing connection to http/2 or sockets)
103 is early hints

I'm not sure how interceptors handle those now if we received them from somewhere. If we ignore them  or what, but maybe we only need to remove them from here as they're not standard request/response style headers that would happen here?

Comment (kettanaito) [reply src/utils/responseUtils.ts]: @mattcosta7, made the status codes a set. Adjusted the test.

**Comment by kettanaito** [comment]
## Released: v0.25.13 🎉

This has been released in v0.25.13!

- 📄 [**Release notes**](https://github.com/mswjs/interceptors/releases/tag/v0.25.13)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/interceptors/v/0.25.13)

Make sure to always update to the latest version (`npm i @mswjs/interceptors@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.

**Pull request merged**